CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ All notable changes to this project will be documented in this file. The format
 - Period progression is now updated when an objective is moved between periods.
 - Fixed a bug causing text to sometimes overflow in tooltips and progress
   history tables.
+- The key result router guard now ensures that the correct period is set.
```

Comment:
Kunne vi vridd på dette til å si noe mer om hva det har å si i appen? At vi har fikset en feil som gjorde at perioden av og til ble vist feil for nøkkelresultater, eller noe sånt?

Reply:
Tenkte på veien hjem at denne kom til å komme, så angret på at det jeg ikke rettet det med en gang (enig altså) 🤣